app/graphql/types/discovery_rule.rb

```diff
@@ -0,0 +1,20 @@
+module Types
```

Comment:
I would move it into `ForemanDiscovery` namespace, so `ForemanDiscovery::Types::DiscoveryRule`